zoop_wrapper/wrapper.py

```diff
@@ -161,13 +161,19 @@ class ZoopWrapper(RequestsWrapper):
         __key: zoop auth token
     """
 
-    def __init__(self):
-        self.__marketplace_id = MARKETPLACE_ID
-        self.__key = ZOOP_KEY
+    BASE_URL = "https://api.zoop.ws/v1/marketplaces/"
 
-        super().__init__(
-            base_url=f"https://api.zoop.ws/v1/marketplaces/" f"{self.__marketplace_id}"
-        )
+    def __init__(self, marketplace_id=None, key=None):
+        if marketplace_id is None:
+            marketplace_id = MARKETPLACE_ID
+
+        if key is None:
+            key = ZOOP_KEY
+
+        self.__marketplace_id = marketplace_id
+        self.__key = key
+
+        super().__init__(base_url=f"{self.BASE_URL}{self.__marketplace_id}")
```

Comment:
Acho que entendi. O import das contantes mudou para dar flexibilidade, se não for fornecido usa o que foi importado aqui, se quem quiser criar no momento de instanciar passar usa o que foi passado, show! Isso faz com que os exemplos fiquem mais didáticos na minha opinião.

Reply:
Sim, pensei justamente neles também